python-package/lightgbm/compat.py

```diff
@@ -11,16 +11,12 @@
 except ImportError:
     PANDAS_INSTALLED = False
 
-    class pd_Series:
+    class pd_Series: # type: ignore
         """Dummy class for pandas.Series."""
 
-        pass
```

Comment:
can you please put these `pass` statements back? I understand that they aren't strictly necessary and that some linters like `pylint` will tell you to remove them, but we have a preference for using `pass` statements in cases like this to make it clear more explicit that the classes don't do anything by themselves.

`mypy` does not care about these, as far as I know, so I don't think the change has an impact on `mypy`'s output